osquery/events/windows/windowseventlogutils.h

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <ctime>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#include <osquery/flags.h>
+#include <osquery/utils/status/status.h>
+
+namespace osquery {
+class WindowsEventLog {
```

Comment:
This class is no longer needed since only the static methods have been extracted

Reply:
Thanks, @alessandrogario for the review. I encapsulate the static functions in `WindowsEventLog` class to avoid name conflict with the `processEvent` and `processEventObject` functions defined for PowerShell events. It will improve the code readability on its usage. Please let me know your thoughts.

Reply:
Classes shouldn't be used purely for namespaces, since C++ does support namespaces natively. I would prefer to remove the class declaration and rename the types, but a standard namespace could work as well

Reply:
Thanks, @alessandrogario for the comments. I removed the class declaration and update the function name.